contents/blog/clickhouse-materialized-columns.md

```diff
@@ -0,0 +1,151 @@
+---
+date: 2021-09-10
+title: How to speed up ClickHouse queries using materialized columns
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: engineering
+author: karl-aksel-puulmann
+featuredImage: ../images/blog/clickhouse-materialized-columns.png
+featuredImageType: full
+---
+
+ClickHouse supports speeding up queries using materialized columns to create new columns on the fly from existing data. In this post, I’ll walk through a query optimization example that's well-suited to this rarely-used feature.
```

Comment:
Just curious what the basis is to say that this feature is rarely used?

Reply:
The documentation is lacking + very little mention of it in the wild :)